sdk/test-utils/test-credential/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.2.0 (2024-06-13)
```

Comment:
update package.json with the same version change please

Reply:
Updated